doc/tutorial/10_grafana.md

```diff
@@ -0,0 +1,36 @@
+# Grafana Dashboard
+
+This tutorial will show you how to encapsulate prometheus standard API and use them to monitor state of pgmoneta with the help of grafana dashboard.
+
+## Preface
+
+This tutorial assumes that you have an installation of PostgreSQL 10+ and pgmoneta.
+
+See [Install pgmoneta](https://github.com/pgmoneta/pgmoneta/blob/main/doc/tutorial/01_install.md)
+for more detail.
+
+## Prometheus Configuration
+
+First of all, you should install prometheus in your pgmoneta server. For `Ubuntu 22.04`, you can just use `apt-get` command to install it.
+
+After you successfully install prometheus, you should replace `prometheus.yml ` with content below (For `Ubuntu 22.04`, the whole path is `/etc/prometheus/prometheus.yml`) to configure how to query your pgmoneta metrics. 
+```yml
+scrape_configs:
+  - job_name: 'pgmoneta'
+    metrics_path: '/metrics'
+    static_configs:
+      - targets: ['localhost:5001']
+```
+
+Then the prometheus service will query your pgmoneta metrics every 15 seconds and package them as time-series data. You can query your pgmoneta metrics and watch their changes as time passed in prometheus web page (default port is `9090`).
+![](https://linzs-upload.oss-cn-shanghai.aliyuncs.com/img/prometheus2.jpg)
+![](https://linzs-upload.oss-cn-shanghai.aliyuncs.com/img/prometheus3.jpg)
+
+## Grafana Dashboard Import
+
+Although prometheus provides capacity of querying and monitoring metrics, we can not customize graph for each metric and place them together.
```

Comment:
prometheus -> Prometheus
graph -> graphs

"and place them together" -> "and provide a unified view"